src/main/resources/ingredients.json

```diff
@@ -45,10 +45,180 @@
   {
     "name": "ham",
     "units": "weight",
-    "unitLabel": "g"
+    "unitLabel": "grams"
   },
   {
     "name": "apple",
     "units": "count"
+  },
+  {
+    "name": "ground chicken",
+    "units": "weight",
+    "unitLabel": "grams"
+  },
+  {
+    "name": "dried basil",
+    "units": "weight",
+    "unitLabel": "grams"
+  },
+  {
+    "name": "dried dill",
+    "units": "weight",
+    "unitLabel": "grams"
+  },
+  {
+    "name": "breadcrumbs",
+    "units": "weight",
+    "unitLabel": "grams"
+  },
+  {
+    "name": "lemon juice",
+    "units": "weight",
+    "unitLabel": "grams"
+  },
+  {
+    "name": "Dijon mustard",
+    "units": "weight",
+    "unitLabel": "grams"
+  },
+  {
+    "name": "pasta",
+    "units": "weight",
+    "unitLabel": "grams"
+  },
+  {
+    "name": "olive oil",
+    "units": "weight",
+    "unitLabel": "grams"
```

Comment:
Perhaps this could be volume to be more consistent with the cooking oil ingredient